swebench/harness/context_manager.py

```diff
@@ -392,7 +393,7 @@ def __enter__(self):
 
                 # Install additional packages if specified
                 if "pip_packages" in install:
-                    cmd = f". {path_activate} {env_name} && pip install {install['pip_packages']}"
+                    cmd = f"source {path_activate} {env_name} && {self.path_conda}/envs/{env_name}/bin/pip install {install['pip_packages']}"
```

Comment:
I had to change this back to using `.`

Reply:
The current version of SWE-bench uses `.`. This has been suggested in a couple prior PRs and we've adopted it.